lang/cs/Org.Apache.REEF.Tang.Tests/Configuration/TestAvroConfiguration.cs

```diff
@@ -62,7 +62,7 @@ private AvroConfiguration ToAvroConfiguration()
             e2.value = "b1=b2";
             b.Add(e1);
             b.Add(e2);
-            var a = new AvroConfiguration(b);
+            var a = new AvroConfiguration("CS", b);
```

Comment:
Would it make sense to have an `enum` defined in Avro for the languages? That would be safer than magic strings, but I don't know how feasible it is.